### DISCUSSION TITLE
Sponsor/Donations?

### DISCUSSION THREAD

**Discussion by RaulPPelaez** [joaotavora/eglot#1332] [General]
Hi! I was wondering about the funding of this amazing project. I was wondering if the authors could intercede and give a few words about how us users can reach and financially support the project. Be it something like Github Sponsors, Patreon or whatever.
Apologies if this information is readily available, a quick search yield nothing -.-

Also, thank you deeply for Eglot, I have used it every day for a long time now and I love it!

**Comment by joaotavora** [comment]
I think I already answered this some years ago. I'd rather keep this volunteer work, for a number of reasons:

* If I start accepting "donations", it changes things at a fundamental level: some people, donors or not, may start -- reasonably -- to feel entitled to my efforts (in fact some already seem to feel like that). I.e. it becomes a disguised and unregulated labor relationship.  I'm not a fan of those, in software or anywhere else for that matter.

* Although I am the author, the coordinator and by some margin the biggest contributor to this project, I am also not the only one. 

So if you or a group of individuals are looking for a specific work (say a specific feature for your company's use case) or think the project could benefit from a secure and firmer investment in maintenance, are willing to compensate me financially for this, you can contact me privately.  Else, if you just want to show appreciation, you can buy me a beer if you're ever in town, and I'll buy the next one :-)

EDIT: you never know what the future holds, and I also have to live and eat and support a family.   So if I'm ever without a job and want to dedicate myself full-time to OSS, I might well change my opinion on this.

**Reply by RaulPPelaez** [reply]
Thanks for the good work and the thorough answer. I understand your reasons.
We live in the same peninsula I believe, so we are not that far for a beer. I owe you one or ten :P 
Feel free to close this one or leave it for reference.

**Reply by joaotavora** [reply]
> We live in the same peninsula I believe, so we are not that far for a beer. I owe you one or ten :P

Great, be sure, to drop me a line if you ever fly from your tourist-infested city down to mine.  I'd love to chat C++ and high-performance-computing while drunk ;-)  My email's in the sources.

**Comment by trev-dev** [comment]
Just wanna throw my name in the hat of gratitude here.  I've watched `eglot` evolve from the scruffy little LSP that could to not only being the official LSP, but honestly, the only one that doesn't suck.

Thank you for making the LSP that performs, is simple to configure, largely dependency free and has sane default behavior.
